holoviews/plotting/plot.py

```diff
@@ -604,6 +604,7 @@ def compute_ranges(self, obj, key, ranges):
         over the whole animation) and finally compute the dimension
         ranges in each group. The new set of ranges is returned.
         """
+        prev_frame = getattr(self, 'prev_frame', None)
```

Comment:
Not a big issue but `self.prev_frame` should probably be initialized to `None` in the appropriate constructor(s)...

Reply:
Don't really want it on all plots, only makes sense on element/overlay plots but this is used even on layout/grid plots.